index.js

```diff
@@ -1,20 +1,13 @@
 'use strict';
 
-const jsonFiles = require('eslint-plugin-json-files');
+const crowdstrikeConfig = require('eslint-config-crowdstrike');
 const globals = require('globals');
-const { FlatCompat } = require('@eslint/eslintrc');
```

Comment:
this is messy due to me alphabetizing. TLDR: nuked the eslintv8 compatibility packages